### PR TITLE
Linked merchant to expense

### PR DESCRIPTION
**Pull request by 1mozolacal**
## Description
Minor changes, expense is now do you can close this branch

## Related Issue
- [ ] This pull request relates to at least one particular issue

  
  - [issue name](https://github.com/ps-toronto-team-4/.github/issues/13)

## Checks
- [X] My pull request adheres to the code style of this project
- [ ] My code requires changes to documentation
- [ ] I have updated the documentation as required
- [X] All the tests have passed

### CODE REVIEW

**Comment by 1mozolacal** [comment]
Hold on I need to add list of Expense to this

File: src/main/resources/graphql/schema.graphqls [hunk 1]
```diff
@@ -46,7 +47,8 @@ type Mutation {
     createMerchant(passwordHash: String!, name: String!, description: String, defaultCategoryId: Int): MerchantPayload!
     updateMerchant(passwordHash: String!, id:Int!, name: String!, description: String, defaultCategoryId: Int): MerchantPayload!
     deleteMerchant(passwordHash:String!, id:Int!): DeleteMerchantPayload!
-    createExpense(passwordHash: String!, title: String!, description: String, amount: Float!): CreateExpensePayload!
+    createExpense(passwordHash: String!, title: String!, description: String, amount: Float!, epochDate: Int, categoryId: Int, merchantId: Int): ExpensePayload!
+    deleteExpense(passwordHash: String!, title: String!, description: String, amount: Float!, epochDate: Int, categoryId: Int, merchantId: Int): ExpensePayload!
```
Comment (jere-mie) [inline src/main/resources/graphql/schema.graphqls:51]: you shouldn't need all of this to delete an expense. You should only need passwordHash and id

File: src/main/resources/graphql/schema.graphqls [hunk 2]
```diff
@@ -90,15 +92,23 @@ union DeleteCategoryPayload = DeleteSuccess | FailurePayload
 type Expense {
 	amount: Float
```
Comment (jere-mie) [inline src/main/resources/graphql/schema.graphqls:93]: we should return the id here as well

**Review by jere-mie — CHANGES_REQUESTED** [review]
make sure to check the requested changes, especially for the graphql schema

**Pull request merged**